### PR TITLE
Removes pg10 support

### PR DESCRIPTION
**Pull request by BradyBonnette**

### CODE REVIEW

**Comment by workingjubilee** [comment]
@BradyBonnette Do you think dropping Postgres 11 as well is worth it?

**Comment by BradyBonnette** [comment]
@workingjubilee tbh I am not sure that is my call. I can certainly remove it here in addition to pg10, but I think I'd want to hear from @eeeebbbbrrrr first.

**Review by Smittyvb — COMMENTED** [review]
[`pgx-pg-sys/include/pg10.h`](https://github.com/tcdi/pgx/blob/brady/remove-pg10/pgx-pg-sys/include/pg10.h) can be deleted now.

**Comment by BradyBonnette** [comment]
@Smittyvb good catch, thanks :) Ill do that right now.

File: cargo-pgx/README.md [hunk 1]
```diff
@@ -504,7 +503,7 @@ test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; fini
 Stopping Postgres
 ```
 
-`cargo pgx test [pg10 | pg11 | pg12 | pg13 | pg14 | pg15]` runs your `#[test]` and `#[pg_test]` annotated functions using cargo's test system.
+`cargo pgx test  pg11 | pg12 | pg13 | pg14 | pg15]` runs your `#[test]` and `#[pg_test]` annotated functions using cargo's test system.
```
[the remark this answers is not in the document]
Comment (BradyBonnette) [reply cargo-pgx/README.md]: 👍

**Review by eeeebbbbrrrr — APPROVED** [review]

**Pull request merged**